applications/NXmx.nxdl.xml

```diff
@@ -688,6 +689,14 @@
 					</attribute>
 				</field>
 			</group>
-			<group type="NXdata" />
+			<group type="NXdata" minOccurs="0">
+				<doc>
```

Comment:
@prjemian this line is under NXentry, is of class NXdata, and can be named anything

Reply:
Understood.  Needed to view the whole file, not just the changes.